### DISCUSSION TITLE
VSC, Feature Request, restarting extension.

### DISCUSSION THREAD

**Discussion by OoLunar** [community/community#12719] [Copilot Conversations]
Hello!

Sometimes my internet isn't very stable. When I try to use Copilot, occasionally I get the following error:

> GitHub Copilot could not connect to server. Extension activation failed: "getaddrinfo EAI_AGAIN api.github.com"

As such, this means I have to restart VSC in order to get Copilot functioning correctly again. Some extensions, such as Omnisharp, have a command to restart their own extension. Would it be realistically feasible to add this to VSC Copilot?

Much appreciation,
\- Avid Copilot enjoyer.

**Comment by bnjunge** [comment]
+1.

**Comment by belkacem759** [comment]
+1

**Comment by ozanerturk** [comment]
+1
